volplugin/mountcount.go

```diff
@@ -0,0 +1,17 @@
+package volplugin
+
+func (dc *DaemonConfig) increaseMount(mp string) int {
+	dc.mountMutex.Lock()
+	defer dc.mountMutex.Unlock()
+
+	dc.mountCount[mp]++
```

Comment:
This state seems to not be stored across daemon restarts.

I've commented to make sure this is tracked.